tests/test_apis.py

```diff
@@ -0,0 +1,209 @@
+"""Test the MyTardis and ICD11 APIs"""
+
+# pylint: disable=redefined-outer-name,invalid-name,protected-access
+from typing import Any, Dict
+
+import mock
+import pytest
+import responses
+from mock import MagicMock
+from requests.models import HTTPError, MissingSchema
+
+from src.ingestion_targets.print_lab_genomics.ICD11_API_agent import ICD11ApiAgent
+from src.ingestion_targets.print_lab_genomics.print_crate_dataclasses import (
+    MedicalCondition,
+)
+from src.mt_api.api_consts import CONNECTION__HOSTNAME
+from src.mt_api.apiconfigs import AuthConfig, MyTardisRestAgent
+
+
+@responses.activate
+def test_mytardis_rest_agent_get(
```

Comment:
Not sure what this test is aiming to do. The response has been mocked so asserting that the status is 200 is not useful here. It does demonstrate that a get request has been made :)

Reply:
Basically all that was for, that a request happens at all. Added a few tests for bad requests to make it more complete.